Makefile

```diff
@@ -243,10 +243,8 @@ build-docs:
 ###                            Docker image                                 ###
 ###############################################################################
 
-build-docker: build-linux
-	cp $(BUILDDIR)/tendermint DOCKER/tendermint
+build-docker:
 	docker build --label=tendermint --tag="tendermint/tendermint" -f DOCKER/Dockerfile .
-	rm -rf DOCKER/tendermint
```

Comment:
Would you mind explaining why this change is ok?

Reply:
The [DOCKER/Dockerfile](https://github.com/tendermint/tendermint/blob/485c96b0d386d5461d397fef76ae75df0d56c885/DOCKER/Dockerfile) is multi-stage and [the fist stage](https://github.com/tendermint/tendermint/blob/485c96b0d386d5461d397fef76ae75df0d56c885/DOCKER/Dockerfile#L1-L8) is building a tendermint binary in docker and [the second](https://github.com/tendermint/tendermint/blob/485c96b0d386d5461d397fef76ae75df0d56c885/DOCKER/Dockerfile#L42) is building resulting docker image with the binary from the previous stage. Because of this, I think that another linux binary build process is redundant, and furthermore built linux binary is not used in the docker image building process.

Reply:
Thanks for the explanation, that definitely makes sense. Thanks for catching that.